Assets/ScriptableObjects/Items/CoilBarrel.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   id: Coil
   displayName: Coil
   secretName: lightning
-  displayDescription: Shoots lightning bolts
+  displayDescription: Shoots bursts of lightning bolts
```

Comment:
We could maybe even say "Shoots 4 bursts of inaccurate lightning bolts"

Reply:
Good suggestion 👍